pydantic/main.py

```diff
@@ -698,10 +698,10 @@ def validate(cls: Type['Model'], value: Any) -> 'Model':
             return cls(**value_as_dict)
 
     @classmethod
-    def _decompose_class(cls: Type['Model'], obj: Any) -> GetterDict:
+    def _decompose_class(cls: Type['Model'], obj: Any, kwargs: Dict[str, Any]) -> GetterDict:
         if isinstance(obj, GetterDict):
             return obj
-        return cls.__config__.getter_dict(obj)
+        return cls.__config__.getter_dict(obj, kwargs)
```

Comment:
This is pretty much a breaking change and should be reflected in docs and history.